boards/riscv/qemu_riscv64/doc/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _qemu_riscv64:
+
+RISCV64 Emulation (QEMU)
+########################
+
+Overview
+********
+
+The RISCV64 QEMU board configuration is used to emulate the RISCV64 architecture.
+
+.. figure:: qemu_riscv64.png
+   :width: 600px
+   :align: center
+   :alt: Qemu
+
+   Qemu (Credit: qemu.org)
+
+Get the Toolchain and Qemu
+**************************
+
+As RISCV64 is not yet included in the Zephyr SDK, it is necessary to
+get a suitable riscv64 toolchain. And since Zephyr does not yet support
```

Comment:
How about instead (note the lines ending with two colons (::) indicate that a code block immediately follows
```
The current `Zephyr SDK v0.10.1 tools
<https://www.zephyrproject.org/developers/#downloads>`_
do not include support for the RISV64 architecture
or for RISC-V floating-point hardware.  You'll need to
configure and compile a toolchain to enable support
for these features, as described here.

Begin by cloning, configuring, and building the toolchain::

   git clone https://github.com/riscv/riscv-gnu-toolchain
   cd riscv-gnu-toolchain
   git submodule update --init --recursive
   ./configure --prefix=$HOME/riscv64-zephyr --with-arch=rv64ima --with-abi=lp64
   make -j4

Next, set Zephyr toolchain environment variables::

  export ZEPHYR_TOOLCHAIN_VARIANT=cross-compile
  export CROSS_COMPILE=$HOME/riscv64-zephyr/bin/riscv64-unknown-elf-